ros2_control_test/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
please use the `test_robot_hardware` package for this.

Reply:
OK!

But, is Idea for you generally sensible? If so I would start to change all test files.

Reply:
Doing this makes a circular dependency with the `hardware_interface` package.

This can be resolved in two ways. adding (this) new package, moving test components plugins to the `hardware_interface` package, or moving the `ResouceManager` to the `controller_manager` package.

@bmagyar, what do you think?

Reply:
Or keeping a very simple set of URDFs directly in `hardware_interface`, even if that means partially duplicating things. Better than shifting too much stuff around IMHO.

Reply:
The actual issue are test plugins for interfaces (Sensor, System, Actutor) if we move those into `hardware_interface` then `test_robot_hardware` package would become the package with test resources which is what we need.
Having these plugins in here is an obsolete structure from ROS1 where an implementation of RobotHW is needed to test every controller. The plugins are not needed anymore for controllers, but only for CM's tests which has already `hardware_interface` as a dependency.
May I create a PR with this?

Reply:
Sounds like a good way to start. Your proposal would then allow us to use/rename test_robot_hw

Reply:
Discussion follows in #208